build64/ldscripts/memtest_shared.lds

```diff
@@ -52,5 +52,4 @@ SECTIONS {
 		. = ALIGN(256);
 		_end = .;
 	}
-	/DISCARD/ : { *(*) }
```

Comment:
This adds ~12KB to the final binary size even without the -Og option, so I'd rather it wasn't done on a non-debug build.

Reply:
It's due to .eh_frame and .comment, the latter being tiny. To keep a single linker script, they could still quit being discarded, but be stripped out individually by `objcopy`.